examples/hf_bert.py

```diff
@@ -0,0 +1,64 @@
+import prewl, json
+
+#-----------------------------------------#
+
+# Optionally use a config.json
+#prewl.configure("config.json")
+
+# Dynamically detect if the argument is json or a file with json
+prewl.configure({
+    'backend': {
+        'service': 'bert-base-uncased',
+        'remote': True,
+        'token': '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' # Use your huggingface token
```

Comment:
This token ok to leave in?

Reply:
Yes, this token is deactivated and expired.